content/en/platform/corda/5.2/deploying-operating/vnodes/upgrade-cpi.md

```diff
@@ -33,10 +33,24 @@ To upgrade a CPI, do the following:
    {{% /tab %}}
    {{% tab name="PowerShell" %}}
    ```shell
-   Invoke-RestMethod -SkipCertificateCheck -Headers @{Authorization=("Basic {0}" -f $REST_API_USER:$REST_API_PASSWORD)} -Method PUT -Uri    $REST_API_URL/api/v5_1virtualnode/<virtualnodeshortid>//cpi/<targetcpifilechecksum>
+   Invoke-RestMethod -SkipCertificateCheck -Headers @{Authorization=("Basic {0}" -f $REST_API_USER:$REST_API_PASSWORD)} -Method PUT -Uri    $REST_API_URL/api/v5_1virtualnode/<virtualnodeshortid>/cpi/<targetcpifilechecksum>
    ```
    {{% /tab %}}
    {{< /tabs >}}
+   {{< enterprise-icon noMargin="true" >}} Alternatively, if you are [using your own virtual node databases]({{< relref "./bring-your-own-db.md" >}}), send the checksum of the CPI to upgrade to using the <a href ="../../reference/rest-api/openapi.html#tag/Virtual-Node-API/operation/get_virtualnode__virtualnodeshortid__db_vault__newcpichecksum_">`api/v5_2/virtualnode/<virtualnodeshortid>/db/vault/<newcpichecksum>` endpoint</a>:
```

Comment:
This step returns SQL that needs to be run on the vnode db. That SQL needs to have been run before running the CPI upgrade request commented on above.

Reply:
Added an extra sentence